packages/wallets/src/features/cashier/modules/CashierLocked/CashierLockedContent.tsx

```diff
@@ -35,167 +81,88 @@ const getCashierLockedDesc = ({
-                </WalletText>
-            ),
-        };
-
-    if (askAuthenticate && poiNeedsVerification) {
-        if (poaNeedsVerification)
-            return {
-                description: (
-                    <WalletText align='center'>
-                        <Trans
-                            components={[
-                                <WalletLink href='/account/proof-of-identity' key={0} variant='bold' />,
-                                <WalletLink href='/account/proof-of-address' key={1} variant='bold' />,
-                            ]}
-                            defaults='Your account has not been authenticated. Please submit your <0>proof of identity</0> and <1>proof of address</1> to authenticate your account and access your {{currency}} Wallet.'
-                            values={{ currency }}
-                        />
-                    </WalletText>
-                ),
-            };
-
-        return {
-            description: (
-                <WalletText align='center'>
-                    <Trans
-                        components={[<WalletLink href='/account/proof-of-identity' key={0} variant='bold' />]}
-                        defaults='Please submit your <0>proof of identity</0> to authenticate your account and access your {{currency} Wallet.'
-                        values={{ currency }}
-                    />
-                </WalletText>
-            ),
-        };
+    if (noResidence) {
+        description = generateDescription(
+            "You've not set your country of residence. To access {{currency}} Wallet, please update your country of residence in the <0>Personal details section</0> in your account settings.",
+            [<WalletLink href='/account/personal-details' key={0} variant='bold' />]
+        );
+    } else if (documentsExpired) {
+        description = generateDescription(
+            'The identification documents you submitted have expired. Please submit valid identity documents to unlock your {{currency}} Wallet.'
+        );
+    } else if (cashierLockedStatus) {
+        description = generateDescription(
+            'Please contact us via <0>live chat</0> to enable deposits and withdrawals again.',
+            [
+                <button
+                    className='wallets-link wallets-link__variant--bold'
+                    key={0}
+                    onClick={() => window.LC_API.open_chat_window()}
+                />,
+            ]
+        );
+    } else if (disabledStatus) {
+        description = generateDescription(
+            'Please contact us via <0>live chat</0> to enable deposits and withdrawals again.',
+            [
+                <button
+                    className='wallets-link wallets-link__variant--bold'
+                    key={0}
+                    onClick={() => window.LC_API.open_chat_window()}
+                />,
+            ]
+        );
```

Comment:
since this two generate the same content could we do this ? 🤔
```
else if (cashierLockedStatus || disabledStatus) {
        description = generateDescription(
            'Please contact us via <0>live chat</0> to enable deposits and withdrawals again.',
            [
                <button
                    className='wallets-link wallets-link__variant--bold'
                    key={0}
                    onClick={() => window.LC_API.open_chat_window()}
                />,
            ]
        );
```

Reply:
Updated